src/timed-event-manager.ts

```diff
@@ -86,10 +84,10 @@ export class TimedEventDisplay extends Phaser.GameObjects.Container {
 
   setup() {
     console.log(this.event?.bannerFilename);
```

Comment:
```suggestion
    this.event = this.event!; // TODO: resolve the `!` issue
    console.log(this.event?.bannerFilename);
```
And then remove all the `!` from the various `this.event` calls after this.

Reply:
Might remove the one from the console then too :D

Reply:
But it would be cleaner to put a proper `this.event` check up instead of just telling the compiler `Yes! it's defined`.

Reply:
Removed the bangs!